### ISSUE TITLE
Stopped working on new GM release

### ISSUE THREAD

**Issue by Dediggefedde** [NuckChorris/AMALL#2]
Firefox revised how extensions are working and what they are allowed to do.
Greasemonkey reacted by introducing a new version, especially rewriting their GM-functions.
There is no backwards-compatibility aside from a polyfil-script.
All GM-functions now are asynchronous and return promise objects.

I made an adaption of the amall-script on my mirror, but the main problem is, that the script is not executed in the iframe it creates, when you push the button. This could be a limitation for the new firefox-extensions... 

I take a closer look into this issue the next days. if I can't make it work like it worked before, I propose to show the llama-givepopup instead and let the user click...

**Comment by NuckChorris** [comment]
Truthfully I didn’t know this script even still worked, that’s amazing. I haven’t used deviantART in ages but I’m more than happy to take fixes

**Comment by Dediggefedde** [comment]
I think I got a solution.
It works again in the new Greasemonkey at Firefox and also in Chrome using Tampermonkey.
Sending the Llama is a bit slower (2-3s) now.
I'm not sure this can be speed up.

I switched from iframe- to embed-tag, since at the moment, iframes are not supported by Greasemonkey. And since jquery embed.load() does not work in Firefox, I switched to embed.get(0).onload. 
Also I tried to add some arrow-functions to clarify use of this objects.

There is also a functional update now: all buttons for the user you just gave a llama to will automatically switch yellow immediately.

@NuckChorris Well... at the moment I'm only in deviantart to maintain my scripts there. ^^
